tests/ui/maybe-bounds.stderr

```diff
@@ -1,22 +1,31 @@
-error: `?Trait` is not permitted in supertraits
+error[E0658]: `?Trait` is not permitted in supertraits
   --> $DIR/maybe-bounds.rs:1:11
    |
 LL | trait Tr: ?Sized {}
    |           ^^^^^^
    |
    = note: traits are `?Sized` by default
+   = help: add `#![feature(more_maybe_bounds)]` to the crate attributes to enable
+   = note: this compiler was built on YYYY-MM-DD; consider upgrading it if it is out of date
```

Comment:
Should such diagnostic changes be visible only on nightly?

Reply:
It's already only visible on nightly according to the logic in `fn add_feature_diagnostics_for_issue`.